Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ghcr.io/astral-sh/uv:python3.10-bookworm-slim
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y \
+    gcc g++ make cmake pkg-config libffi-dev libssl-dev \
+    libjpeg-dev libpng-dev libfreetype6-dev libxml2-dev \
+    libxslt1-dev zlib1g-dev curl \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV UV_HTTP_TIMEOUT=900 \
+    UV_CONCURRENT_DOWNLOADS=32 \
+    UV_CONCURRENT_BUILDS=8 \
+    UV_CACHE_DIR=/tmp/uv-cache \
+    UV_NO_PROGRESS=true \
+    UV_COMPILE_BYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    PYTHONDONTWRITEBYTECODE=1 \
+    PIP_NO_CACHE_DIR=1 \
+    PIP_DISABLE_PIP_VERSION_CHECK=1 \
+    PYTHONPATH=/app \
+    MILVUS_DATA_PATH=/app/data
+
+RUN mkdir -p /tmp/uv-cache /app/data /app/logs
+
+COPY pyproject.toml uv.lock LICENSE README.md ./
+COPY deepsearcher/ ./deepsearcher/
+
+RUN uv sync --frozen --no-dev && \
```

Comment:
delete `&& \  uv pip install -e .` because `uv sync ...` is the installation command